pkg/asset/machines/worker.go

```diff
@@ -115,7 +115,7 @@ func defaultAzureMachinePoolPlatform() azuretypes.MachinePool {
 
 func defaultGCPMachinePoolPlatform() gcptypes.MachinePool {
 	return gcptypes.MachinePool{
-		InstanceType: "n1-standard-4",
+		InstanceType: "n2-standard-4",
```

Comment:
I don't think we need to change this one. `n1-standard-4` is close enough in terms of specs.

Reply:
Yeah I thought so too but wanted another opinion

Reply:
I would agree with @patrickdillon the 1GB difference in memory is more than likely not noticed in a default configuration.

Reply:
Spoke to Mak about this and he wanted this set to n2-standard-4.